src/lib/Scenes/Show/Components/Shows/Components/ShowItem.tsx

```diff
@@ -1,127 +1,79 @@
-import { Flex } from "@artsy/palette"
+import { Flex, Sans, Serif } from "@artsy/palette"
 import { ShowItem_show } from "__generated__/ShowItem_show.graphql"
-import { WhiteButton } from "lib/Components/Buttons"
 import OpaqueImageView from "lib/Components/OpaqueImageView"
-import { Colors } from "lib/data/colors"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
 import React from "react"
-import { Text } from "react-native"
+import { Dimensions, TouchableOpacity } from "react-native"
 import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components/native"
 
-const DEFAULT_ARTWORK_URL = "https://d32dm0rphc51dk.cloudfront.net/ADYdY8P1kg9vKA9ffrb4pg/larger.jpg"
-
+const { width: windowWidth } = Dimensions.get("window")
```

Comment:
This width could change, right? We might want to setup a listener to handle device rotations: https://facebook.github.io/react-native/docs/dimensions#addeventlistener

Reply:
Yeah, you can listen for onLayout's too I think - that's the pattern elsewhere

Reply:
Yes, that would only be relevant for iPad landscape and portrait orientations and we should just pick a card width that works for both in this case